CODEOWNERS

```diff
@@ -1 +1,116 @@
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/inbound/ @Arshardh @arunans23 @SajinieKavindya @senthuran16 @dushaniw
+
+# Area : Websub, SSE
+/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/WebsubSubscriptionConfiguration.java @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/test/resources/repository/resources/api_templates/websub_api_template.xml  @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/webhooks/  @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/webhooks/  @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/webhooks/  @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/mediators/webhooks/  @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/streaming/webhook/ @senthuran16  @chanikag
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/streaming/sse/  @senthuran16  @chanikag
+
+# Area : GraphQL, GraphQL Complexity
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/graphQL/ @HiranyaKavishani @ashera96
+/components/apimgt/org.wso2.carbon.apimgt.common.gateway/src/main/java/org/wso2/carbon/apimgt/common/gateway/graphql/ @HiranyaKavishani @ashera96
+/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/graphql/ @HiranyaKavishani @ashera96
+/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/gateway/GraphQLSchemaDTO.java  @HiranyaKavishani @ashera96
+/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/GraphQLSchemaDefinition.java @HiranyaKavishani @ashera96
+
+# Area : GraphQL Subscriptions
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/inbound/websocket/GraphQLProcessorResponseDTO.java @wasuradananjith @dushaniw
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/inbound/websocket/response/GraphQLResponseProcessor.java @wasuradananjith @dushaniw
+/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/inbound/websocket/request/GraphQLRequestProcessor.java @wasuradananjith @dushaniw
+
+# Area : Gateway Artifact Management/Synchronization
+/components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/gatewayartifactsynchronizer/ @1akshitha @Krishanx92
```

Comment:
@1akshitha should be changed to @lakshithagunasekara

Reply:
@tharikaGitHub we may need to revise this codeowners again. I will add a new pr for this.